easybuild/easyblocks/s/superlu.py

```diff
@@ -0,0 +1,133 @@
+##
+# Copyright 2016 University of Luxembourg
```

Comment:
AFAIK, this should either be dual-licensed MIT/GPL or Kenneth will ask you for extra work :)
(dual licensing is cheap and is based on precedent though)